src/containers/Login/Login.module.scss

```diff
@@ -43,6 +43,8 @@
     margin: auto;
     cursor: pointer;
     text-decoration: none;
+    max-width: 120px;
+    margin-top: 0;
```

Comment:
I think you can remove `margin-top` and even `margin` properties

Reply:
instead of adding max-width and margin-top, I just removed margin: auto which has the same effect